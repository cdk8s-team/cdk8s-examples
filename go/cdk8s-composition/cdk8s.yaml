app: go run .
language: go
imports:
  - k8s@1.26.0