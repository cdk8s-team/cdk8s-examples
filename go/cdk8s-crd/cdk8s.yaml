app: go run .
language: go
imports:
  - k8s@1.26.0
  - https://raw.githubusercontent.com/cert-manager/cert-manager/master/deploy/crds/crd-certificates.yaml