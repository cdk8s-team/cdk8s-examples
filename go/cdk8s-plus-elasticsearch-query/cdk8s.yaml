app: go run .
language: go
imports:
  - k8s@1.26.0
  - https://download.elastic.co/downloads/eck/1.1.2/all-in-one.yaml