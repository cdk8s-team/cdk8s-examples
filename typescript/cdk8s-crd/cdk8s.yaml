app: ts-node index.ts
language: typescript
imports:
  - https://raw.githubusercontent.com/cert-manager/cert-manager/master/deploy/crds/crd-certificates.yaml