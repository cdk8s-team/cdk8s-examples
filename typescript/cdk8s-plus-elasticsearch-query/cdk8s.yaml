app: ts-node index.ts
language: typescript
imports:
  - k8s@1.26.0
  - https://download.elastic.co/downloads/eck/1.1.2/all-in-one.yaml