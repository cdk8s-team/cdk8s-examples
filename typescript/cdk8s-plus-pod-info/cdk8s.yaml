app: ts-node index.ts
language: typescript
