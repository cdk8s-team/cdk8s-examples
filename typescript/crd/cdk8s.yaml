app: ts-node index.ts
language: typescript
imports:
  - k8s@1.26.0
  - https://raw.githubusercontent.com/cert-manager/cert-manager/master/deploy/crds/crd-certificates.yaml