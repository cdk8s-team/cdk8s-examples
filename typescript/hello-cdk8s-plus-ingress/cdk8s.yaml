app: ts-node index.ts
language: typescript
imports:
  - k8s@1.26.0