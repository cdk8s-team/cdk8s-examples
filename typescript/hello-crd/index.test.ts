import { HelloCrd } from './index';
import { Testing } from 'cdk8s';

test('snapshot', () => {
  const app = Testing.app();
  const chart = new HelloCrd(app, 'hello-crd');

  expect(Testing.synth(chart)).toMatchSnapshot();
});