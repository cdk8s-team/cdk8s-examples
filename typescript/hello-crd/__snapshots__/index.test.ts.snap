// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
[
  {
    "apiVersion": "cert-manager.io/v1",
    "kind": "Certificate",
    "metadata": {
      "name": "hello-crd-certificate-c8b00ace",
    },
    "spec": {
      "issuerRef": {
        "name": "issuer",
      },
      "secretName": "secret",
    },
  },
]
`;
